editing/update-attributes/src/main/java/com/esri/samples/update_attributes/UpdateAttributesSample.java

```diff
@@ -153,12 +158,12 @@ public void start(Stage stage) {
                 GeoElement element = identified.get(0);
                 // get selected feature
                 if (element instanceof ArcGISFeature) {
-                  selected = (ArcGISFeature) element;
-                  featureLayer.selectFeature(selected);
-                  selected.loadAsync();
-                  selected.addDoneLoadingListener(() -> {
-                    if (selected.getLoadStatus() == LoadStatus.LOADED) {
-                      comboBox.getSelectionModel().select((String) selected.getAttributes().get("typdamage"));
+                  selectedFeature = (ArcGISFeature) element;
+                  featureLayer.selectFeature(selectedFeature);
```

Comment:
Perhaps `identifiedFeature` would be a better name? Looks odd to select the selected feature.